index.html

```diff
@@ -504,7 +504,12 @@ <h2>
         at least its <a>origin</a>, including whether it is served over a
         secure connection. This UI SHOULD differ from any UI used when the
         [=Document/URL=] is <a>within scope</a>, in order to make it obvious
-        that the user is navigating off scope.
+        that the user is navigating off scope. In addition, the user agent
+        SHOULD reset the <a>application context</a>'s <a>default theme
+        color</a> to a user agent-defined default value. The off-scope document
+        may override that color through the inclusion of a valid [[HTML]]
```

Comment:
You can't use the word "may" without it being in all-caps, making it a normative requirement. And you can't impose a normative requirement on the document; all requirements are imposed on the user agent.

"The user agent MAY use the HTML meta element whose name attribute is "theme-color" of the off-scope document to theme the application context."